x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ml/integration/MlRestTestStateCleaner.java

```diff
@@ -41,7 +39,7 @@ private void deleteAllDatafeeds() throws IOException {
         final Response datafeedsResponse = adminClient.performRequest(datafeedsRequest);
         @SuppressWarnings("unchecked")
         final List<Map<String, Object>> datafeeds =
-                (List<Map<String, Object>>) XContentMapValues.extractValue("datafeeds", testCase.entityAsMap(datafeedsResponse));
+                (List<Map<String, Object>>) XContentMapValues.extractValue("datafeeds", ESRestTestCase.entityAsMap(datafeedsResponse));
```

Comment:
Checkstyle forced me to change this and the rest of the changes to this file come about because of the change here.